test/test_dataloader.py

```diff
@@ -67,7 +67,8 @@
 # as well during the execution of this test suite, and it will cause
 # CUDA OOM error on Windows.
 TEST_CUDA = torch.cuda.is_available()
-
+dev_name = torch.cuda.get_device_name(torch.cuda.current_device()).lower()
```

Comment:
Would this crash, if no GPU is available, or what is `cuda.get_device_name` returning in this case?

Reply:
yes it would crash, i added a guard against this.